packages/serialize/README.md

```diff
@@ -16,12 +16,12 @@
 
 ## Description
 
-A serialize utility to be able to convert nodejs data types into a json compatible format.
+Do you want serialize Node.js data types to a JSON compatible format? Then this is the package for you!
```

Comment:
Lad, maybe something a bit more descriptive haha

Reply:
Not sure how much more descriptive it gets lmao